docker/README.txt

```diff
@@ -0,0 +1,16 @@
+
+Containerized deployment for CFE-18. Requires external database to use. It uses public tomcat docker image (tomcat:8-jre8-temurin-focal)
+
+How to use?
```

Comment:
We will build and provide our own container images so this should be done in usage and not self-build/development point of view

Example on how it has been done in other project https://github.com/teragrep/lsh_01?tab=readme-ov-file#how-to-run-in-a-container